code/game/objects/items.dm

```diff
@@ -398,56 +398,6 @@ GLOBAL_DATUM_INIT(welding_sparks, /mutable_appearance, mutable_appearance('icons
 			R.activate_module(src)
 			R.hud_used.update_robot_modules_display()
 
-// Due to storage type consolidation this should get used more now.
-// I have cleaned it up a little, but it could probably use more.  -Sayu
-/obj/item/attackby(obj/item/I, mob/user, params)
-	if(isstorage(I))
-		var/obj/item/storage/S = I
-		if(S.use_to_pickup)
```

Comment:
Just to make sure, did you test to make sure bags like chem/ore bags properly picked up stacks of the items they can carry?

Reply:
Also tested this, it works properly